extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -618,12 +622,8 @@ private function extendSections( $curatedContent ) {
 			$section['items'] = $this->extendItemsWithImages( $section['items'] );
 			$section['items'] = $this->extendItemsWithType( $section['items'] );
 			return $section;
-		}, $curatedContent );
+		}, $data );
 
-		$data = array_values( array_filter( $data, function ( $section ) use ( $validator ) {
-			$error = isset( $section['featured'] ) ? false : $validator->validateSection( $section );
```

Comment:
It was introduced just 26 days ago, I suppose there was a reason :thinking:
@idradm please take a look.